devops/scripts/register_all_bundles_in_repo.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -eu
```

Comment:
```suggestion
set -o errexit
set -o pipefail
set -o nounset
# Uncomment this line to see each command for debugging (careful: this will show secrets!)
# set -o xtrace
```